packages/react-next/src/components/OverflowSet/OverflowSet.test.tsx

```diff
@@ -33,38 +32,26 @@ function getPersistedKeytip(keytipManager: KeytipManager, keySequences: string[]
   return ktp ? ktp.keytip : undefined;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const noOp = (): any => undefined;
```

Comment:
so many `noOp` implemented everywhere :) perhaps move this to our test utils? 
Don't have to be in this PR

Reply:
I think it's a simple enough thing that I'd rather just declare it in each file as needed. Kind of like how you probably don't need the `leftpad` npm package. :P